bot/exts/utilities/realpython.py

```diff
@@ -31,9 +32,16 @@ def __init__(self, bot: Bot):
 
     @commands.command(aliases=["rp"])
     @commands.cooldown(1, 10, commands.cooldowns.BucketType.user)
-    async def realpython(self, ctx: commands.Context, *, user_search: str) -> None:
-        """Send 5 articles that match the user's search terms."""
-        params = {"q": user_search, "limit": 5, "kind": "article"}
+    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str) -> None:
+        """
+        Send `amount` articles that match the search terms (`user_search`).
+
+        `amount` must be between 1 and 5 (inclusive). If it's not, an error message is sent.
+        """
```

Comment:
I'm not too fond of having annotated doc strings like this for commands, since this is the help that will be given to the user I think it should read as "proper" English. Maybe something more like this
```suggestion
        """
        Send some articles from RealPython that match the search terms.

        By default the top 5 matches are sent, this can be overwritten to
        a number between 1 and 5 by specifying an amount before the search query.
        """
```